aii-ks/src/main/pan/quattor/aii/ks/config.pan

```diff
@@ -84,6 +84,10 @@ variable AII_OSINSTALL_PROTOCOL = value('/system/aii/osinstall/ks/osinstall_prot
 #
 variable AII_OSINSTALL_ROOT ?= undef;
 variable AII_OSINSTALL_OS_VERSION ?= undef;
+variable DEBUG = debug(format('%s: AII_OSINSTALL_ROOT=%s, AII_OSINSTALL_OS_VERSION=%s',
+                                OBJECT,
+                                to_string(AII_OSINSTALL_ROOT),
```

Comment:
remove the `to_string`, format does that just fine

Reply:
Was not aware, will fix it...